ResourceRest/PayloadAliasRegistry.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\ResourceRestBundle\ResourceRest;
```

Comment:
I'm -1 for such duplicated namespaces, we should move this to the root of the bundle instead in that case imo